endpoints/openrtb2/video_auction.go

```diff
@@ -189,6 +189,9 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 	if bidReq.App != nil {
 		labels.Source = pbsmetrics.DemandApp
 		labels.PubID = effectivePubID(bidReq.App.Publisher)
+		if _, found := deps.cfg.BlacklistedAppMap[labels.PubID]; found {
+			return
```

Comment:
See above somment on simply returning at this point.

Reply:
Seen and done